### PR TITLE
CST Updates and Fixes

### PR DESCRIPTION
**Pull request by vader111**
Fix CST Nosecone Model and fix up a bunch of issues.

### CODE REVIEW

**Comment by vader111** [comment]
Will Also Need https://github.com/Standecco/ROKerbalism/pull/80

**Comment by vader111** [comment]
(Ready To Review)

File: GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg [hunk 1]
```diff
@@ -234,52 +246,6 @@ PART
...
-		ProgressFxModules = 1
-
-		TRANSMITTER
-		{
-			PacketInterval = 1.0
-			PacketSize = 1.024
-			PacketResourceCost = 0.01
-		}
-	}
-}
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTCM.cfg:278]: Can clean up the patches, but do not strip RemoteTech configs.

File: GameData/ROCapsules/PartConfigs/CST/CSTHS.cfg [hunk 2]
```diff
@@ -36,29 +53,45 @@ PART
...
-	
-	tags = starliner, boeing, cst100, cst-100, 100
-	
+
+	heatShieldDiameter	= 4.56
 	heatShieldTag = LEO
-	heatShieldDiameter = 4.56
-	resetHeatShieldMass = true
 	resetHeatShieldAblator = true
+	resetHeatShieldMass = true
```
Comment (DRVeyl) [inline GameData/ROCapsules/PartConfigs/CST/CSTHS.cfg:65]: This formatting is not better ;)   (tab insertion?)

**Review by DRVeyl — APPROVED** [review]

**Pull request merged**